e2e-tests/gef/cypress/fixtures/helpers.js

```diff
@@ -0,0 +1,5 @@
+const toTitleCase = (text) => text[0].toUpperCase() + text.substring(1).toLowerCase();
```

Comment:
Is there a test for this?

Reply:
technically yes, since without it, the e2e tests fail for submission confirmation panel :P but no real proper test of it since in e2e-tests folder

Reply:
Sorry didn't realise it was in e2e test 😄